app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -544,6 +364,48 @@ class TopicFragmentTest {
     return launch(createTopicActivityIntent(internalProfileId, topicId))
   }
 
+  private fun clickTabAtPosition(position: Int) {
+    onView(
+      Matchers.allOf(
+        withText(TopicTab.getTabForPosition(position).name),
+        isDescendantOfA(withId(R.id.topic_tabs_container))
+      )
+    ).perform(click())
+  }
+
+  private fun matchTabTitleAtPosition(position: Int) {
+    onView(withId(R.id.topic_tabs_container)).check(
+      matches(
+        matchCurrentTabTitle(
+          TopicTab.getTabForPosition(position).name
+        )
+      )
+    )
+  }
+
+  private fun matchStringOnListItem(
+    recyclerView: Int,
+    position: Int,
```

Comment:
Perhaps: ``itemPosition`` would be clearer.

Reply:
done